### PR TITLE
Add SQL utils class

### PR DESCRIPTION
**Pull request by NatSamo**
- Integrated Microsoft SQL Server database into the project.
- Configured environment variables for database connection using .env.
- Implemented logic for connecting to the database and executing SQL queries using the mssql module.
- Updated tests to verify database connection and query execution.

### CODE REVIEW

File: database/db.ts [hunk 1]
```diff
@@ -0,0 +1,40 @@
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+// src/database/db.ts
```
Comment (gbaranov) [inline database/db.ts]: Remove comments plz

File: .env
Comment (gbaranov) [inline .env:1]: Please add this to .gitignore

File: database/db.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+interface DBConfig {
+  user: string;
+  host: string;
+  database: string;
+  password: string;
+  port: number;
+}
+
+export class DB {
+  //private DBConfig: DBConfig;
```
Comment (gbaranov) [inline database/db.ts]: No commented code plz

File: tests/database.test.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+//import { fixtures } from '../fixtures/dbFixture';
```
Comment (gbaranov) [inline tests/database.test.ts]: Clean comment

File: tests/fixtures/dbFixture.ts [hunk 4]
```diff
@@ -0,0 +1,44 @@
+import { beforeEach, afterEach } from 'node:test';
+import { DB } from '../../database/db';
+
+// Конфигурация подключения к базе данных
```
Comment (gbaranov) [inline tests/fixtures/dbFixture.ts]: Especially Russian ones

File: tests/fixtures/dbFixture.ts [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+const dbConfig = {
+  user: process.env.DB_USER || 'your_user',
+  host: process.env.DB_HOST || 'localhost',
+  database: process.env.DB_NAME || 'test_db',
```
Comment (gbaranov) [inline tests/fixtures/dbFixture.ts]: OR values can be removed

**Review by gbaranov — DISMISSED** [review]

File: database/db.ts [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+export class DB {
+  async executeQuery(query: string, config) {
+    const client = new ConnectionPool(config);
+    try {
+      await client.connect();
+      const result = await client.query(query);
+      console.log(result.rows);
+      return result.rows;
+    } catch (error) {
+      console.error("Ошибка подключения/выполнения запроса:", error);
```
Comment (Elgrueber) [inline database/db.ts]: would you please replace with english "Error during connection/request" or smth like this

**Review by Elgrueber — APPROVED** [review]

**Pull request merged**